_meta/beat.yml

```diff
@@ -178,6 +178,10 @@ apm-server:
   #ilm:
     #enabled: false
 
+  # When using agent remote configuration, information fetched from Kibana will be cached in memory for some time.
+  # Specify cache key expiration via this setting. Default is 30 seconds.
+  #remote_config.agent.cache.expiration: 30s
```

Comment:
have you considered _not_ introducing a user-facing setting?

Reply:
Yes I have. I am a bit twisted on whether or not it is necessary, but went for more flexibility with it. I am also ok with removing this if you don't think it is needed. 
@graphaelli any opinion?

Reply:
might as well keep it

Reply:
revisiting, can we drop the remote from this - how about `agent_config.cache.expiration`

Reply:
I changed to `agent.config.cache.expiration` in case we need to nest additional configs in the future.